.github/workflows/ci.yml

```diff
@@ -161,39 +215,56 @@ jobs:
           TF_FILE_LOCATION: deploy/aws/app/kube
           TF_BACKEND_ARGS: region=${{ secrets.AWS_TF_STATE_REGION }},access_key=${{ secrets.AWS_ACCESS_KEY_ID }},secret_key=${{ secrets.AWS_SECRET_ACCESS_KEY }},bucket=${{ secrets.AWS_TF_STATE_BUCKET }},key=${{ secrets.AWS_TF_STATE_KEY }},dynamodb_table=${{ secrets.AWS_TF_STATE_DYNAMOTABLE }},encrypt=${{ secrets.AWS_TF_STATE_ENCRYPTION }}
           # Terraform Resource Configuration
-          TF_VAR_env:  "nonprod"
+          TF_VAR_env: "nonprod"
           TF_VAR_owner: ${{ env.OWNER }}
           TF_VAR_region: ${{ env.REGION }}
           TF_VAR_docker_image_name: ${{ env.DOCKER_IMAGE_NAME }}
           TF_VAR_stage: "nonprod"
           TF_VAR_name_company: ${{ env.COMPANY }}
           TF_VAR_name_project: ${{ env.PROJECT }}
-          TF_VAR_name_domain: "{{ env.DOMAIN }}"
+          TF_VAR_name_domain: ${{ env.DOMAIN }}
+          # Helm Chart Configuration
+          HELM_CHART_FILE: "deploy/helm/stacks-dotnet"
+          HELM_VALUE_FILE: "deploy/helm/stacks-dotnet/values.yaml"
```

Comment:
Shouldn't need these as both are defined at the top level

Reply:
Oh OK, I'm not sure how they ended up duped here, I'll remove thanks

Reply:
Removed